### ISSUE TITLE
Hangouts on Air

### ISSUE THREAD

**Issue by mikeumus** [bevry-archive/meta#1]
_The following post is maintained by the community:_

Hey everyone! We occasionally run online meetings for the Bevry community, as a chance to discuss the latest going on and to move things forward together.
## Bevry Team Setup Guide
1. Logging into the Bevry Google+ Page
2. Going to [this url](https://plus.google.com/b/100631142988286661025/hangouts)
3. Clicking the `Start a Google Hangout on Air` button
4. Creating a Google Drive Document in [this folder](https://drive.google.com/#folders/0B6MqiLy7C3PhNmNHS0U1OUFMYU0)
5. Set the Google Drive Document to be editable by all
6. Use the shareable link of the Google Drive Document as the Hangout on Air meeting notes
7. Give it the name `Bevry Catchup #XX` where XX is the number we are up to
8. In the audience field have "Public", "DocPad", and "Bevry"
9. Click start
## Participant Guide

[Google Hangouts on Air](https://www.google.com/+/learnmore/hangouts/onair.html) is the medium we're using for these daily catchups; it is quite full featured, allowing for a remote virtual meeting, streaming of the meeting, and saving to YouTube afterwards.

**Join our [Bevry](https://plus.google.com/communities/105289334765050598053) and [DocPad](https://plus.google.com/communities/102027871269737205567) Google+ Community Pages**. We will post the invitations to the hangouts on them.

Each invitation will allow you to watch the Google Hangout on Air and ask questions. Also posted with the invitation will be a link to a document that we will be using to plan the catchup and take notes during it. Contained inside the document will be a link to join as a caller within the Google Hangout on Air.

[You can watch our previous catchups here.](https://www.youtube.com/channel/UC0HLq-UQ49GA43aHbxJlXNg)

**Comment by RobLoach** [comment]
Bi-weekly on #Docpad might be a good start. If we just start _something_, we can iterate to improve it over time. http://doodle.com is a handy tool to find when people are available.

**Comment by RobLoach** [comment]
I added three different time slots in EST. We'd likely have to change the times to accomodate our vast time differences, however: http://doodle.com/wi7xf4m5itg3gbnk

Feel free to add your times so that we could schedule this. It's based on the next couple weeks.

**Comment by mikeumus** [comment]
Doodle done.

Hey @pflannery, when's good for you? Use the Doodle link above!

**Comment by balupton** [comment]
All replied. So let's proceed with that 11am (sydney time) slot for me?

Then perhaps you guys can do your own with @stongo for the 6am slot?

**Comment by mikeumus** [comment]
Works for me :+1:

**Comment by balupton** [comment]
Catchup 1 a job well done :-)

[Youtube](https://www.youtube.com/watch?v=Ejzc7V65Feg&list=UUDh-MCltU7QCrFvXMt3p8jw)
[Google Doc](https://docs.google.com/document/d/160tEIzOq4jNkABVEqrrueRULTUoXAE29sucerDXFyS8/edit)

**Comment by mikeumus** [comment]
:+1:  :+1:

**Comment by stongo** [comment]
Thanks for sharing the stream. Watched the first bit and will watch the 
rest later, but here's an initial thought
I agree with Ben that abstracting actions is the best first step 
forward. Second to that, being able to register new actions would 
compliment this. That way plugins, including servers for example, could 
register their own action, as Ben hinted as "express" for base server. 
As a final inclusion, a method to list actions in docpad class and as a 
cli command would be cool.

**Comment by mikeumus** [comment]
@stongo, yeah expanding the CLI would be cool. We could add a lot of power there. 

Also Ben and I were chatting about using/expanding @RobLoach's DocPad Yeoman generator so that we don't have skeletons and plugins breaking all the time. Less work for us. This sounds like a priority for me: https://github.com/RobLoach/generator-docpad

I'm focusing on the new DocPad homepage and it's new marketing messages. I need to know more about the awesomeness that is the taskgroup, bal-util, joe so that I can speak succinctly on them. I'll already be speaking on the approach that DocPad is taking by having everything be small pieces and products of their own and the ability to swap out so much of DocPad. How it can be a SSG and also a robust platform for dynamic prototyping as well. I'll create a new issue to discuss marketing message now. In the Google Doc Ben linked above, we have loomio linked as a tool to help us make decentralized decisions for many of the issues we're about to face as we have many choices ahead of us in the immediate future: https://github.com/loomio/loomio

I'll work on setting this up as well.

**Comment by RobLoach** [comment]
@mikeumus @balupton Apologies for not being able to make it. I was caught off-guard and didn't expect it to be the Monday of. You two missed having my support, and I definitely am on board for contributing all that's possible.

Well done on the conversation. I found it extremely motivating and was great to get a bit more context into where everyone would love to see things go. I found myself agreeing with all your ideas, and was really caught up with what could happen. Both your willingness to hop on and have such a driven topic discussion was absolutely amazing to see.

Let's set up individual issues for these to keep discussions focused and on-going. I put together the Generator discussion: https://github.com/RobLoach/generator-docpad/issues/22

**Comment by mikeumus** [comment]
Honestly Ben caught me a little off guard as well when he asked me if I was ready on Skype but this brings up an important topic as well of doing things now and not lagging at all. Especially on a huge important project like DocPad where so many people benefit. Ben knows, and I agree, that the more voice/round-table meetings we have (daily) the more productive we'll be as a team and the sooner we'll find our DocPad groove. None of us are sure what that will look like exactly yet but the more we get together in those hangouts the sooner well find out.

**Comment by balupton** [comment]
Let's just lock them in as daily, starting at 11am Sydney time for organisation, and kicking off at 11.30am each day on YouTube. The more we do them, the shorter they'll be.

**Comment by balupton** [comment]
I've updated the original post with the latest info. Let's use further comments on this issue only for feedback relating to to the process of the hangouts.

**Comment by stongo** [comment]
Sorry I can't make the 11am AEST time-slot. I guess I should be trying to watch the youtube streams and post commends in Hangout group?

**Comment by balupton** [comment]
@stongo no worries. I noticed the only slot you could do is the 6am Sydney time slot. I'm pretty sure @mikeumus and @RobLoach are happy doing that time in addition, the frequency is up to them. But totally cool to have multiple catchup sessions.

**Comment by balupton** [comment]
For everyone, I'm considering making a new Google account that we can use for the hangouts. So that way, if I'm not available, another owner can still proceed with adding a video to the official account.

**Comment by mikeumus** [comment]
@balupton, making an account like that would allow me to come in after and do some post-production as well with YouTube's video popovers etc...  We should be posting videos on a official Bevry channel anyways.

**Comment by balupton** [comment]
Cool. Created #5 for discussion about that.

One thing to add to our agenda is refine our priorities inside workflowly, and get the others setup with it.

Personally, I feel our big technical priorities are:
- fixing bugs — @RobLoach ???
- abstracting out plugin system into [plugin-engine](https://github.com/bevry/plugin-engine) — @balupton
- unifying the websites — @balupton @mikeumus 
- new blog — @mikeumus 
- new DocPad site — @mikeumus 

Then phase 2 being:
- yeoman generator official — @RobLoach 
- update docs for yeoman generator — @RobLoach @greduan
- deprecate any skeletons better served by the yeoman generator — @RobLoach @mikeumus  @greduan
- implement 336 — @RobLoach @balupton ?
- new startup hostel website — @balupton

Then phase 3 being:
- interconnect — @balupton
- exporters and better importers — @balupton
- more dissembling — @RobLoach 
- more tests — @mikeumus @RobLoach  ???

They should be able to all be done in parallel given we can share the load effectively.

**Comment by balupton** [comment]
New time poll https://doodle.com/z9rc9ri4pusrd6fe

**Comment by mikeumus** [comment]
As per today's Catchup5 Hangout, we have the idea to mix up the topics of our hangouts so that everyday isn't just shop talk. Ideas are as follows:

_Moved to: https://github.com/bevry/meta/issues/12#issuecomment-42093164_

**Comment by balupton** [comment]
I will be unavailable to run or participate in the catchup sessions until the Thursday of next week. If someone else can take over during period that would be great!

**Comment by greduan** [comment]
I was wondering why there weren't any catchup sessions in these last couple of days.

I personally can't organize these, mainly because of my internet speed, and I don't feel like I'm qualified. lol

**Comment by greduan** [comment]
Something we can discuss maybe as an option? https://waffle.io/

**Comment by mikeumus** [comment]
We've already dicussed and tried Waffle.io. It was clunky and the UI/UX isn't the best.
@huboard was what we favored: https://huboard.com/bevry/docpad

p.s. Just got back from DC, ready for doing Catchups again.

**Comment by balupton** [comment]
Next hangout on air is scheduled. Details:
https://github.com/bevry/meta/issues/18#issuecomment-55257577

**Comment by pflannery** [comment]
Can't make this as it will be 3am here in London. I will replay ;)

**Comment by balupton** [comment]
Migrated to: https://discuss.bevry.me/t/hangouts-on-air/38
